com.unity.render-pipelines.high-definition/Runtime/Lighting/LightCookieManager.cs

```diff
@@ -223,12 +223,55 @@ public void LayoutIfNeeded()
             }
         }
 
+        public Vector4 Fetch2DCookie(CommandBuffer cmd, Texture cookie, Texture ies)
+        {
+            int width  = (int)Mathf.Max(cookie.width, ies.height);
+            int height = (int)Mathf.Max(cookie.width, ies.height);
+
+            if (width < k_MinCookieSize || height < k_MinCookieSize)
+                return Vector4.zero;
+
+            if (!m_CookieAtlas.IsCached(out var scaleBias, m_CookieAtlas.GetTextureID(cookie, ies)) && !m_NoMoreSpace)
+                Debug.LogError($"2D Light cookie texture {cookie} can't be fetched without having reserved. You can try to increase the cookie atlas resolution in the HDRP settings.");
```

Comment:
Don't think ti is the correct workding
"2D Light cookie texture {cookie} can't be fetched without having reserved. You can try to increase the cookie atlas resolution in the HDRP settings."
@JordanL8 could you check? thanks

Reply:
How about:
"Unity cannot fetch the 2D Light cookie texture: {cookie} because it is not on the cookie atlas. To resolve this, open your HDRP Asset and increase the resolution of the cookie atlas."

Reply:
This message is the one we use on the current version.
I can change it with this one if it makes more sense.